server.py

```diff
@@ -818,6 +829,14 @@ def get(self, region, id):
 
         return return_dict
 
+class SmashGgMappingResource(restful.Resource):
```

Comment:
let's change this to SmashGGMappingResource to be consistent with SmashGGScraper